### DISCUSSION TITLE
any EIP-191 signed_data can never be an Ethereum transaction

### DISCUSSION THREAD

**Discussion by arifulone** [Cyfrin/security-and-auditing-full-course-s23#143] [Q&A] [answered]
https://eips.ethereum.org/EIPS/eip-191

In this page says , 
```
We propose the following format for signed_data

0x19 <1 byte version> <version specific data> <data to sign>.
The initial 0x19 byte is intended to ensure that the signed_data is not valid RLP.

For a single byte whose value is in the [0x00, 0x7f] range, that byte is its own RLP encoding.

That means that any signed_data cannot be one RLP-structure, but a 1-byte RLP payload followed by something else. Thus, any EIP-191 signed_data can never be an Ethereum transaction.


```

i couldn't understand it , 
can u guys explain it ?

**Comment by EngrPips** [comment, marked as answer]
Hello @arifulone , so what they are trying to say is that the `EIP 191` intentionally  made it a standard to prefix any message that would be signed using the `EIP 191` standard with `0x19` so that it doesn't have the same structure as the normal transaction signed on ethereum. basically they created a different between what is an ethereum signed message and an ethereum signed transaction by prefixing an ethereum signed message with `0x19`.
